install/pre-install-payload/scripts/reqs-deploy.sh

```diff
@@ -87,6 +110,27 @@ function uninstall_containerd_artefacts() {
 	fi
 }
 
+function uninstall_nydus_snapshotter_artefacts() {
```

Comment:
Where is this called from?

Reply:
This should've been called from `uninstall_artifacts`, but I missed it.
Very nice catch!